runtime/compiler/build/toolcfg/gnu/common.mk

```diff
@@ -537,12 +545,9 @@ ifneq ($(JITSERVER_SUPPORT),)
 
         ifneq ($(OPENSSL_CFLAGS),)
             ADD_OPENSSL_CFLAGS=false
-            # --with-openssl=system, OPENSSL_LIBS looks like "-L/path/to/system/openssllib -lssl -lcrypto"
-            ifneq ($(OPENSSL_LIBS),)
-                SOLINK_LIBPATH+=$(subst -L,,$(firstword $(OPENSSL_LIBS)))
-                ADD_OPENSSL_CFLAGS=true
             # --with-openssl=fetched --enable-openssl-bundling
-            else ifneq ($(OPENSSL_BUNDLE_LIB_PATH),)
+            ifneq ($(OPENSSL_BUNDLE_LIB_PATH),)
+                SOLINK_SLINK+=crypto
```

Comment:
`# --with-openssl=system, OPENSSL_LIBS looks like "-L/path/to/system/openssllib -lssl -lcrypto"`

How do both `ssl` and `crypto` get included in the link command line now?

Reply:
`--with-openssl=system` should be handled as the same as the current default case when `--with-openssl` is not set: JITServer is built against the system installed OpenSSL. libssl is linked to the system installed libcrypto. 

--with-openssl=system or no flag is set. 
```
$SOLINK_SLINK is [j9jit_vm j9codert_vm j9util j9utilcore j9pool j9avl j9stackmap j9hashtable j9jit_vm j9codert_vm j9util j9utilcore j9pool j9avl j9stackmap j9hashtable j9thr29 j9hookable29 dl m ssl]

$SOLINK_LIBPATH is [/root/home/ahuo/src/openj9-openjdk-jdk8/build/linux-x86_64-normal-server-release/vm/compiler/.. /root/home/ahuo/src/openj9-openjdk-jdk8/build/linux-x86_64-normal-server-release/vm/compiler/../lib]

$SOLINK_FLAGS is [ -m64 -static-libgcc -static-libstdc++ -Wl,-S ]

>ldd libj9jit29.so
libssl.so.1.1 => /usr/lib/x86_64-linux-gnu/libssl.so.1.1 (0x00007f8d3f5f5000)
libcrypto.so.1.1 => /usr/lib/x86_64-linux-gnu/libcrypto.so.1.1 (0x00007f8d3e9ee000)
```

--with-openssl=fetched
```
$SOLINK_SLINK is [j9jit_vm j9codert_vm j9util j9utilcore j9pool j9avl j9stackmap j9hashtable j9jit_vm j9codert_vm j9util j9utilcore j9pool j9avl j9stackmap j9hashtable j9thr29 j9hookable29 dl m ssl]

$SOLINK_LIBPATH is [/root/home/ahuo/src/openj9-openjdk-jdk8/build/linux-x86_64-normal-server-release/vm/compiler/.. /root/home/ahuo/src/openj9-openjdk-jdk8/build/linux-x86_64-normal-server-release/vm/compiler/../lib /root/home/ahuo/src/openj9-openjdk-jdk8/openssl]

$SOLINK_FLAGS is [ -m64 -static-libgcc -static-libstdc++ -Wl,-S ]

>ldd libj9jit29.so
libssl.so.1.0.0 => /usr/lib/x86_64-linux-gnu/libssl.so.1.0.0 (0x00007fd63b23e000)
libcrypto.so.1.0.0 => /usr/lib/x86_64-linux-gnu/libcrypto.so.1.0.0 (0x00007fd63a66c000)
```

--with-openssl=fetched --enable-openssl-bundling
```

$SOLINK_SLINK is [j9jit_vm j9codert_vm j9util j9utilcore j9pool j9avl j9stackmap j9hashtable j9jit_vm j9codert_vm j9util j9utilcore j9pool j9avl j9stackmap j9hashtable j9thr29 j9hookable29 dl m ssl crypto]

$SOLINK_LIBPATH is [/root/home/ahuo/src/openj9-openjdk-jdk8/build/linux-x86_64-normal-server-release/vm/compiler/.. /root/home/ahuo/src/openj9-openjdk-jdk8/build/linux-x86_64-normal-server-release/vm/compiler/../lib /root/home/ahuo/src/openj9-openjdk-jdk8/openssl]

$SOLINK_FLAGS is [ -m64 -static-libgcc -static-libstdc++ -Wl,-S  -Wl,-rpath,\$ORIGIN/..]

>ldd libj9jit29.so
libssl.so.1.0.0 => /root/home/ahuo/src/openj9-openjdk-jdk8/build/linux-x86_64-normal-server-release/images/j2sdk-image/jre/lib/amd64/compressedrefs/../libssl.so.1.0.0 (0x00007f91d40f7000)
libcrypto.so.1.0.0 => /root/home/ahuo/src/openj9-openjdk-jdk8/build/linux-x86_64-normal-server-release/images/j2sdk-image/jre/lib/amd64/compressedrefs/../libcrypto.so.1.0.0 (0x00007f91d3c93000
```